src/inmanta/protocol/methods_v2.py

```diff
@@ -764,42 +765,49 @@ def resource_logs(
-                    `?filter.timestamp=ge:2021-08-18T09:21:30.568353&filter.timestamp=lt:2021-08-18T10:21:30.568353`.
-                    Multiple constraints can be specified, in which case only log messages that match all constraints will be
-                    returned.
-                message: filter by the content of the log messages. Partial matches are allowed. (case-insensitive)
-                minimal_log_level: filter by the log level of the log messages. The filter specifies the minimal level,
-                so messages with either this level, or a higher severity level are going to be included in the result.
-                For example, for `filter.minimal_log_level=INFO`, the log messages with level `INFO, WARNING, ERROR, CRITICAL`
-                all match the query.
-    :param sort: Return the results sorted according to the parameter value.
-                It should follow the pattern `<attribute_to_sort_by>.<order>`, for example `timestamp.desc`
-                (case insensitive).
-                The only sorting by `timestamp` is supported.
-                The following orders are supported: 'asc', 'desc'
+        Filters should be specified with the syntax `?filter.<filter_key>=value`, for example `?filter.minimal_log_level=INFO`.
+        It's also possible to provide multiple values for the same filter, in this case resources are returned, if they match
+        any of these filter values.
+
+        For example: `?filter.action=pull&filter.action=deploy` returns logs with either of the actions pull or deploy.
+        Multiple different filters narrow the results however (they are treated as an 'AND' operator).
+        For example `filter.minimal_log_level=INFO&filter.action=deploy` returns logs with 'deploy' action, where the
+        'log_level' is at least 'INFO'.
+
+        The following options are available:
+
+            action: filter by the action of the log
+
+            timestamp: return the logs matching the timestamp constraints. Valid constraints are of the form
+            "<lt|le|gt|ge>:<x>". The expected format is YYYY-MM-DDTHH:mm:ss.ssssss, so an ISO-8601 datetime string,
+            in UTC timezone.
+
+            For example: `?filter.timestamp=ge:2021-08-18T09:21:30.568353&filter.timestamp=lt:2021-08-18T10:21:30.568353`.
+            Multiple constraints can be specified, in which case only log messages that match all constraints will be
+            returned.
+
+            message: filter by the content of the log messages. Partial matches are allowed. (case-insensitive)
+
+            minimal_log_level: filter by the log level of the log messages. The filter specifies the minimal level,
+            so messages with either this level, or a higher severity level are going to be included in the result.
+
+            For example, for `filter.minimal_log_level=INFO`, the log messages with level `INFO, WARNING, ERROR, CRITICAL`
+            all match the query.
+
```

Comment:
This also renders rather weird in the API documentation. I am also not sure if we can do something about it.
![Screenshot from 2023-01-17 13-37-19](https://user-images.githubusercontent.com/2684622/212900979-6d784086-ba0e-4706-a17e-30ec670df7d7.png)

Reply:
I added some bullet points and the programmatic API reference now looks like this:

![image](https://user-images.githubusercontent.com/98876549/213182398-492c8acd-7482-4297-8121-a012d45ff3b8.png)

I haven't check how the API doc itself renders, I don't know/remember how to generate it

Reply:
Nice!

Reply:
You can render the documentation using `make docs`.

Reply:
Yes, that's what I used and then checkout the output in docs/build/html/index.html but it doesn't render the same on my end so I thought the API documentation you were mentioning was something else.